packages/babel-parser/src/parser/statement.js

```diff
@@ -2536,19 +2536,17 @@ export default class StatementParser extends ExpressionParser {
       node.key = this.parseIdentifier(true);
 
       if (node.key.name !== "type") {
-        this.raise(
-          Errors.ModuleAttributeDifferentFromType,
-          { node: node.key },
-          node.key.name,
-        );
+        this.raise(Errors.ModuleAttributeDifferentFromType, {
+          at: node.key,
+          // key: node.key.name,
```

Comment:
I had originally commented it because I was confused why it was there, it later turned out that it was just not used (this is the one I mention in the summary). I have now gotten rid of it though, good catch!